mscore/timesigwizard.ui

```diff
@@ -361,6 +361,50 @@
      </layout>
     </widget>
    </item>
+   <item>
+    <widget class="QGroupBox" name="tempoGroup">
+     <property name="accessibleName">
+      <string>Tempo</string>
+     </property>
+     <property name="title">
+      <string>Tempo</string>
+     </property>
+     <property name="checkable">
+      <bool>true</bool>
+     </property>
+     <property name="checked">
+      <bool>false</bool>
+     </property>
+     <layout class="QHBoxLayout">
+      <item>
+       <widget class="QLabel" name="labelBpm">
+        <property name="text">
+         <string>BPM:</string>
+        </property>
+       </widget>
+      </item>
+      <item>
+       <widget class="QDoubleSpinBox" name="spinboxTempo">
+        <property name="accessibleName">
+         <string>Beats per minute</string>
+        </property>
+        <property name="minimum">
+         <number>20</number>
+        </property>
+        <property name="maximum">
+         <number>400</number>
+        </property>
+        <property name="decimals">
+         <number>1</number>
+        </property>
+        <property name="value">
+         <number>120</number>
+        </property>
+       </widget>
+      </item>
```

Comment:
Probably need to add a horizontal space here

        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>